LongWarOfTheChosen/Config/XComLW_Overhaul.ini

```diff
@@ -597,14 +597,14 @@ INSTANT_BUILD_TIMES=TRUE
 ;Mag
 +ItemTable=(ItemTemplateName="Bullpup_MG",	Slots=3,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="MagnetizedWeapons",		RequiredTech2="",	SupplyCost=40,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="",	SpecialItemCost=0,	TradingPostValue=20, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_Common.MagSMG_Base")
 +ItemTable=(ItemTemplateName="VektorRifle_MG",	Slots=3,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="MagnetizedWeapons",		RequiredTech2="",	SupplyCost=50,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="",	SpecialItemCost=0,	TradingPostValue=25, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_Common.MagVektor_Base")
-+ItemTable=(ItemTemplateName="ShardGauntlet_MG",	Slots=0,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="PlatedArmor",		RequiredTech2="AutopsyAdventStunLancer",	SupplyCost=35,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="CorpseAdventStunLancer",	SpecialItemCost=1,	TradingPostValue=15, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_StrategyImages.Inv_MagTGauntlet")
```

Comment:
Is it OK to have no Tech1 requirement but have a Tech2 requirement? I genuinely don't know, so want to make sure it's OK.

Reply:
T1 do not have tech requirements as they are available from the start with no research. Also removed the armor requirements for t2.